scripts/handlers.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-console */
+const { globSync } = require('glob10');
+const frontmatterGH = require('@github-docs/frontmatter');
```

Comment:
we have a `scripts/utils/frontmatter.js` module that can handle parsing frontmatter from a file. it doesn't have a stringify currently, but i'd rather we add it there than pull in another package

Reply:
ooh gotcha, this is just from the original script